File: app/src/main/java/com/lzm/lightLive/http/request/hy/HyConverter.java
package com.lzm.lightLive.http.request.hy;

import android.util.Log;
import com.lzm.lightLive.http.CustomGsonConvertFactory;
import java.io.IOException;
import okhttp3.ResponseBody;

public class HyConverter extends CustomGsonConvertFactory.ResponseConverter {

    @Override
    public String convertResponse(ResponseBody value) throws IOException {
        String valueStr = value.string();
        Log.e("convertResponse: ", valueStr );
        return valueStr;
    }

}

File: app/src/main/java/com/lzm/lightLive/http/request/hy/HyConverter.kt
package com.lzm.lightLive.http.request.hy

import android.util.Log
import com.lzm.lightLive.http.CustomGsonConvertFactory.ResponseConverter
import okhttp3.ResponseBody
import java.io.IOException

class HyConverter : ResponseConverter() {
    @Throws(IOException::class)
    public override fun convertResponse(value: ResponseBody): String {
        val valueStr = value.string()
        Log.e("jsonConverter: ", valueStr)
        return valueStr
    }
}